index.html

```diff
@@ -0,0 +1,135 @@
+         </nav>
+         </div>
+         <div class="header2">
+          
+          <nav class="nav">
+             <div class="logo">
+              <a href="index.html"> <img class="logo-img1" src="images/logo.png" alt="logo cc Global summit 2015" width="100" height="50"></a>
+               </div>
+               <figure class="humburger">
+                <span  class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+               </figure>
+               <ul class="nav-items" id="nav-items">
+                <li><a href="" class="closebtn">x</a></li> 
+                <li><a href="index.html" class="nav-item nav-actives">Home</a> </li>
+                <li><a href="About.html" class="nav-item">About</a> </li>
+                <li><a href="#program" class="nav-item">Program</a> </li>
+               <li><a href="#locate" class="nav-item">Sponsor</a> </li>
+               <li><a href="#locate" class="nav-item CcChampaing">Cc Champaing</a> </li>
+             </ul>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <section class="invitation">
+          <h3 class="hello-text">"WelCome ToWorld Conference Alerts"</h3>
+          <h1 class="tittle tittleHome">  WORLD CONFERENCE <br> ALERTS 2022, <br>2023 </h1>
+          <div class="invite-wrap">
+            <p class="invite-msg tittleHome">International Conference on Climate Change aims to bring together  scientists, researchers and research scholars to exchange and share their experiences and research results.</p>
+          </div>
+          <h2 class="date-info1">22nd Nov 2022 To 23rd Nov 2022</h2>
+          <p class="locate1">@ International Conference on Climate Change (ICCC)</p>
+        </section>
+        <section class="main-program " id="program">
+          <h2 class="secondary-tittle ">Main Program</h2>
+          <div class="bar-icon"></div>
+          <div class="program-list">
+            <div class="program">
+              <img class="program-icon" src="images/program_icon_01.png" alt="">
+              <h3 class="program-tittle">Lecture</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+            </div>
+            <div class="program " >
+              <img class="program-icon" src="images/program_icon_02.png" alt="">
+              <h3 class="program-tittle program-tittleNoWrap">CC Exhibition</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+            </div>
+            <div class="program ">
+              <img class="program-icon" src="images/program_icon_03.png" alt="">
+              <h3 class="program-tittle">Forum</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
```

Comment:
- [ ] You did a great job implementing the changes requested by the previous reviewer! 💯 🥇 However, these changes introduced a new bug for the mobile version:

![image](https://user-images.githubusercontent.com/77069011/203858571-e84bce91-04e7-4daf-a810-b0cce7033819.png) 

Make sure to have a working mobile design until at least, `375px`, although I would recommend extending this value to `360px` the reason behind these measurements is the number of people worldwide that has devices this wide. [Here are the numbers](https://worship.agency/mobile-screen-sizes-for-2022-based-on-data-from-2021). Please fix this text overlapping! 🤓 🛠️